cmd/detectExecuteScan.go

```diff
@@ -44,8 +44,17 @@ func addDetectArgs(args []string, config detectExecuteScanOptions) []string {
 	args = append(args, fmt.Sprintf("--blackduck.url=%v", config.ServerURL))
 	args = append(args, fmt.Sprintf("--blackduck.api.token=%v", config.APIToken))
 
-	args = append(args, fmt.Sprintf("--detect.project.name=%v", config.ProjectName))
-	args = append(args, fmt.Sprintf("--detect.project.version.name=%v", config.ProjectVersion))
+	args = append(args, fmt.Sprintf("--detect.project.name=\\\"%v\\\"", config.ProjectName))
+	args = append(args, fmt.Sprintf("--detect.project.version.name=\\\"%v\\\"", config.ProjectVersion))
```

Comment:
please verify in a test that you get the expected output

Reply:
The test has been updated to make sure the output is as expected